definitions/variable/technology/power-plant.yaml

```diff
@@ -129,86 +127,44 @@
-    description: Efficiency of pumping for a pumped-storage hydro unit
-    unit: '%'
-    skip-region-aggregation: true
-- Turbine Efficiency|Electricity|Hydro|Pumped Storage:
-    description: Efficiency of turbining for a pumped-storage hydro unit
-    unit: '%'
-    skip-region-aggregation: true
 - Load Factor|Electricity|{Variable Renewable Power Type}:
-    description: Load factors for onshore wind power plants
+    description: Load factors for {Variable Renewable Power Type} power plants
     unit: '%'
     skip-region-aggregation: true
-- Maximum Energy Charge|Electricity|Energy Storage System:
-    description: This is the upper bound that energy storage system could charge energy
-      at time "t".
-    unit: GWh
-- Maximum Charge|Electricity|Energy Storage System:
-    description: Maximum charge when storing energy the energy storage system unit.
-    unit: GW
-- Roundtrip Efficiency|Electricity|Energy Storage System|{Energy Storage System Type}:
+- Roundtrip Efficiency|Electricity|{Electricity Storage Type}:
     description: Efficiency when charging energy from the grid to the system.
-    unit: 
-- Maximum Discharge|Electricity|Energy Storage System:
-    description: Maximum discharge when consuming energy from the energy storage system
-      unit.
-    unit: GW
-- Maximum Storage|Electricity|Energy Storage System:
-    description: Maximum energy that can be stored by the energy storage systems.
-    unit: GWh
-- Maximum Storage|Electricity|Energy Storage System|{Energy Storage System Type}:
-    description: Maximum energy that can be stored by the Lithium Iron Batteries systems.
-    unit: GWh
-- Minimum Storage|Electricity|Energy Storage System:
-    description: Minimum energy that can be stored by the energy storage system unit.
-    unit: GWh
-- Spillage|Electricity|Energy Storage System:
-    description: Energy spillage that could not be stored by an energy storage system
-      (ESS). Several energy models include spillage variables related to renewable
-      energy production; however, another way is to include a spillage variable to
-      the ESS unit since it is the last sink of the power network.
-    unit: GWh
-- Seasonality|Electricity|Energy Storage System|{Energy Storage System Type}:
-    description: seasonality (daily, weekly, monthly, etc.) of {Energy Storage System Type}
-    unit: [daily,weekly,monthly]
-- Seasonality|Electricity|Hydro|Reservoir:
-    description: seasonality (daily, weekly, monthly, etc.) of Reservoir
-    unit: [daily,weekly,monthly]
-- Seasonality|Electricity|Hydro|Pumped Storage:
-    description: seasonality (daily, weekly, monthly, etc.) of Pumped Storage
-    unit: [daily,weekly,monthly]
+    unit:
+- Storage Type|Electricity|{Electricity Storage Type}:
+    description: Storage type (daily, weekly, monthly, etc.) of a {Electricity Storage Type} unit. It means that the unit can be defined as 1 when it is daily, 7 when it is weekly, 30 when it is monthly, etc.
+    unit: days
```

Comment:
I thought we had agreed to remove this variable which is not compliant with the general rules (as its value cannot be a number, which is mandatory) and to keep the variable Seasonnality which unit can be either hours, days, months and its value is a number (of days, of months, of years) thus giving the same information as the 'storage type' (which name is moreove highly ambiguous also), and can also allow more flexibility

Reply:
Yeah, I checked all of it. I found that the branch needed to be rebased. I've just done it.

Reply:
I rebased my local branch to synchronize with the current version of the main branch. So, this seasonally variable is already in this PR.